agir/groups/components/groupPage/GroupSettings/GroupMemberPage.js

```diff
@@ -46,19 +46,17 @@ const GroupMemberPage = (props) => {
       <Spacer size="2rem" />
 
       {members.map((e, id) => (
-        <>
+        <Fragment key={id}>
```

Comment:
Pas bloquant : 
c'est mieux d'utiliser l'id (`e.id`) du membre comme `key` plutôt que l'index (cf. https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318 ).

D'ailleurs , pourquoi `e` et `id` comme noms de paramètres, plutôt que `member` et `i` ou `index` ?

Reply:
En effet on en avait parlé, je change ça !